footprints/main/views.py

```diff
@@ -317,8 +317,45 @@ def get_rows(self):
                    'Imprint Actor and Role', 'Imprint BHB Number',
                    'Imprint OCLC Number', 'Evidence Type', 'Evidence Location',
                    'Evidence Call Number', 'Evidence Details']
+        for r in Role.objects.for_footprint():
+            role = 'Footprint Role ' + unicode(r.name)\
```

Comment:
Also, i think all the calls to `.encode('utf-8')` should be removed, because it looks like that will double-encode the string:
```
In [2]: smart_text('ábá')
Out[2]: u'\xe1b\xe1'

In [3]: smart_text('ábá').encode('utf-8')
Out[3]: '\xc3\xa1b\xc3\xa1'
```